pkgs/misc/drivers/logiops/0_3_3.nix

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
This one should be default.nix as it's the latest version. (and presumably will be updated with upstream releases)